contributing/release-schedule.md

```diff
@@ -1,84 +1,154 @@
+
+There are two sets of build configurations used in creating release builds: the _canary_ configuration and the _production_ configuration. The **experimental** and **beta** release channels are built off of the same commit, but with the _canary_ and _production_ configurations, respectively. The _canary_ configuration enables experimental components and features that may be turned off in _production_. Each is served to 0.5% of AMP traffic. It is possible to opt into the **experimental** or **beta** channels via the `AMP_CANARY` cookie; see the [Opt-In Cookie section](#opt-in-cookie) for details.
+
+The **stable** release is built with the _production_ configuration and served to most AMP traffic. Since the **beta** release is also built from the _production_ configuration, it represents the exact build which will become **stable** the following week (ignoring the possibility of [cherry-picks](#cherry-picks)).
+
+An additional **control** release channel exists, and is also served to 0.5% of AMP traffic. This release contains the same binaries as **stable**, and is used as a traffic-eqivalent baseline to compare **experimental** and **beta** against for metrics and error reporting. It cannot be opted into.
+
+See the [release cadence section](#release-cadence) for a more detailed breakdown of the release promotion cadence. To determine if a PR has been included in the current _weekly_ release, look for the GitHub labels _PR Use: In Canary_ or _PR Use: In Production_.
+
+### LTS
+
+The **lts** release channel provides a previous **stable** build for four-month intervals. Every four weeks, the current **stable** release is promoted to **lts**. This is not recommended for all AMP partners, but is provided so partners performing a QA cycle may do so less often. The **lts** release must be explicitly opted into by adding `/lts` to the path of the runtime and extension scripts requested in the page:
+
+> Note: The LTS release channel is available only to AMP HTML pages; it is not available to AMP Actions, AMP for Email, AMP Stories, or AMP for Ads.
+
+```html
+<!-- Standard AMP HTML runtime and extension scripts -->
+<script async src="https://cdn.ampproject.org/v0.js"></script>
+<script async custom-element='amp-ad'
+     src='https://cdn.ampproject.org/v0/amp-ad-0.1.js'></script>
+
+<!-- LTS AMP HTML runtime and extension scripts -->
+<script async src="https://cdn.ampproject.org/lts/v0.js"></script>
+<script async custom-element='amp-ad'
+     src='https://cdn.ampproject.org/lts/v0/amp-ad-0.1.js'></script>
+```
+
+As long as all runtime and extension scripts in a page must use the same release version, requesting the LTS scripts is considered valid AMP and gets the same cache benefits as the **stable** release channel.
```

Comment:
❌ this paragraph - this is something for the LTS doc

Reply:
Moved.